Library/Homebrew/test/language/node_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "language/node"
+
+describe Language::Node do
+  specify "#npm_cache_config" do
```

Comment:
I don't think that behaviour needs tested at all; it's an implementation detail.